cl/opinion_page/views.py

```diff
@@ -297,7 +297,7 @@ async def view_parties(
         .order_by("name", "party__name")
     )
 
-    parties = {}
+    parties: Dict[str, list] = {}
```

Comment:
Just say 
```
from collections import defaultdict
.
.
.

parties = collections.defaultdict(list)
async for party_type in party_types:
    parties[party_type.name].append(party_type)
````

Reply:
Yeah, I saw this one too, but didn't want to stress since things are broken right now. Want to put this in your next PR?

Reply:
Sure.